Sources/PostgresNIO/New/PostgresRowSequence.swift

```diff
@@ -26,6 +26,21 @@ public struct PostgresRowSequence: AsyncSequence, Sendable {
             columns: self.columns
         )
     }
+
+    /// Get the column information of the query results.
+    public func getColumns() -> [PostgresColumn] {
+        self.columns.map { column in
+            PostgresColumn(
+                name: column.name,
+                tableOID: column.tableOID,
+                columnAttributeNumber: column.columnAttributeNumber,
+                dataType: column.dataType,
+                dataTypeSize: column.dataTypeSize,
+                dataTypeModifier: column.dataTypeModifier,
+                format: column.format
+            )
+        }
+    }
```

Comment:
This is an O(n) call. Where `n` equals the number of columns. Do we need a returned Array? Can we return a custom PostgresColumns struct that wraps the existing `[RowDescription.Column]` structure?

Example:
```
struct PostgresColumns: Sequence {
    typealias Element == PostgresColumn
   
    var underlying: [RowDescription.Column]

    func makeIterator() -> Iterator {
        Iterator(underlying: self.underlying.makeIterator)
    }

    struct Iterator: IteratorProtocol {
        var underlying: [RowDescription.Column].Iterator

        func next() -> PostgresColumn {
           self.underlying.next()
        }
    }
}
```

Reply:
If we do the above we can just add a get only property:

```swift
var columns: PostgresColumns {
    PostgresColumns(underlying: self.columns)
}
```